connection/src/test/java/com/netcracker/ConnectionTest.java

```diff
@@ -0,0 +1,42 @@
+package com.netcracker;
+
+import com.netcracker.domain.Message;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.SocketAddress;
+
+import static com.netcracker.domain.MessageType.TEXT;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class ConnectionTest {
```

Comment:
&#x1F34E; **Issue**
Tests in this class do no make sense ;) They are testing themselves.